packages/firebase_messaging/lib/firebase_messaging.dart

```diff
@@ -3,13 +3,41 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
+import 'dart:ui';
 
 import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
 import 'package:meta/meta.dart';
 import 'package:platform/platform.dart';
 
 typedef Future<dynamic> MessageHandler(Map<String, dynamic> message);
 
+const String _backgroundName = 'plugins.flutter.io/android_fcm_background';
```

Comment:
Not sure it makes sense to put this here as it's only used in one place, but should probably have channel in the method name

Reply:
Done